packages/components/tabs/src/tab.ts

```diff
@@ -51,61 +56,55 @@ export class Tab extends LitElement {
     this.slot ||= 'tabs';
   }
 
+  override updated(changes: PropertyValues<this>): void {
+    super.updated(changes);
+
+    if (changes.has('selected')) {
+      this.setAttribute('aria-selected', Boolean(this.selected).toString());
+    }
+  }
+
   override render(): TemplateResult {
-    return this.href
-      ? html`<a href=${this.href}>${this.renderContent()}</a>`
-      : html`<div class="wrapper">${this.renderContent()}</div>`;
+    return this.href && !this.disabled
+      ? html`<a href=${this.href} part="outer" role="presentation" tabindex="-1">${this.renderContent()}</a>`
+      : html`<div part="outer">${this.renderContent()}</div>`;
   }
 
   /** @ignore */
   renderContent(): TemplateResult {
     return html`
-      <slot @slotchange=${this.#onIconSlotChange} name="icon" part="icon"></slot>
-      <div class="content">
-        <span class="title">
-          <slot @slotchange=${this.#onSlotChange}></slot>
-          <slot name="badge" part="badge"></slot>
-        </span>
+      <div part="inner">
+        <slot @slotchange=${this.#onIconSlotChange} name="icon" part="icon"></slot>
+        <slot @slotchange=${this.#onSlotChange} part="title"></slot>
+        <slot name="badge" part="badge"></slot>
         <slot @slotchange=${this.#onSubtitleSlotChange} name="subtitle" part="subtitle"></slot>
       </div>
     `;
   }
 
-  override updated(changes: PropertyValues<this>): void {
-    super.updated(changes);
-
-    if (changes.has('selected')) {
-      this.setAttribute('aria-selected', this.selected ? 'true' : 'false');
-    }
-  }
-
   #onIconSlotChange(event: Event & { target: HTMLSlotElement }): void {
     const hasIcon = event.target.assignedElements({ flatten: true }).length > 0;
 
     this.toggleAttribute('has-icon', hasIcon);
   }
 
   #onKeydown(event: KeyboardEvent): void {
-    if (!this.href || this.disabled) {
-      return;
-    } else if (event.key === 'Enter' || event.key === ' ') {
+    if (this.disabled) {
       event.preventDefault();
+      event.stopPropagation();
 
-      // Enter automatically triggers a click event, but space does not
-      this.renderRoot.querySelector<HTMLElement>('a[href]')?.click();
```

Comment:
On the website we have tabs with links, right now when I click on the tab it works fine, the link is being activated, but it's not working when I press `space` or `enter`

<img width="1035" alt="image" src="https://github.com/user-attachments/assets/78636ce8-6f84-479a-beb4-f568b2a2c2f9" />

Reply:
And it looks like the border in the `part="container"` is missing on the website:

<img width="968" alt="image" src="https://github.com/user-attachments/assets/9f313d6c-0104-4071-8d0d-46cc70959046" />